index.html

```diff
@@ -4,8 +4,82 @@
      <meta charset="UTF-8">
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      <title>project 3</title>
+     <link rel="stylesheet" href="style.css">
+   <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Kanit:wght@300&display=swap" rel="stylesheet">
  </head>
  <body>
-     
+     <header>
+       <h1>
+        Design conference<br>
+         ——Typography In Web Design
+       </h1>
+      </header>
+       <nav>
+      <ul class="navigation" >  
+      <li><a href="#">Home</a></li>
+      <li><a href="#">Speakers</a></li>
+      <li><a href="#">Register</a></li>
+      <li><a href="#">Contact</a></li>
+      </ul> 
+      </nav>
+   <main>
+     <ul class="u-list-reset ">
+    <li class="talk1">
+     <img src="https://cdn.glitch.com/0e85a349-5917-49b5-9721-e8ec891917e8%2F1.png?v=1607879901347"
+        alt="Photo of Jordan Devos" />
+     <h3>Jordan Devos</h3>
```

Comment:
Here, you're using an `h3` element, but there is no `h2` element in your markup. You want to avoid skipping heading levels, so you should either increase this heading or consider adding a `h2` above this, maybe a heading for "Talks"?